kernel/src/expressions/mod.rs

```diff
@@ -40,6 +40,8 @@ pub enum BinaryOperator {
 pub enum VariadicOperator {
     And,
     Or,
+    AndKleene,
+    OrKleene,
```

Comment:
The default in arrow is when either input is null then the output is null

Reply:
Yeah. I discussed internally. Spark and the Java kernel use "kleene" style, so I think this can be greatly simplified by just only having that, and documenting that that's the behavior